mlkem/native/aarch64/intt_clean.S

```diff
@@ -314,16 +358,13 @@ layer123_start:
         gs_butterfly data2, data6, root0, 0, 1
         gs_butterfly data3, data7, root0, 0, 1
 
+        // Bounds: < 8q
+
         str q_data4, [in, #(4*(512/8))]
         str q_data5, [in, #(5*(512/8))]
         str q_data6, [in, #(6*(512/8))]
         str q_data7, [in, #(7*(512/8))]
 
-        // Scale half the coeffs by 1/n; for the other half, the scaling has
-        // been merged into the multiplication with the twiddle factor on the
-        // last layer.
-        mul_ninv data0, data1, data2, data3, data0, data1, data2, data3
```

Comment:
So you've removed this code, and adjusted the twiddle factors back to their original values?

Reply:
The code moved to the top.